### PR TITLE
docs(deploy): add node cluster mode

### PR DESCRIPTION
**Pull request by Mastercuber**
### ❓ Type of change

- [x] 📖 Documentation (updates to the documentation or readme)
- [ ] 🐞 Bug fix (a non-breaking change that fixes an issue)
- [ ] 👌 Enhancement (improving an existing functionality like performance)
- [ ] ✨ New feature (a non-breaking change that adds functionality)
- [ ] ⚠️ Breaking change (fix or feature that would cause existing functionality to change)

### 📚 Description

This PR adds a section to the docs for starting load balanced network processes with a Node.js internal module.

### 📝 Checklist

- [ ] I have linked an issue or discussion.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

**Comment by pi0** [comment]
Hi @Mastercuber and thanks for nice PR. I've made https://github.com/unjs/nitro/issues/451 to introduce cluser as a nitro preset. Would you like to help implementing that?

**Comment by Mastercuber** [comment]
Yes I would like to.
I made me some thoughts about how could a cluster module looks like, but maybe it's better to switch to the issue in [unjs](https://github.com/unjs/nitro/issues/451).

**Comment by pi0** [comment]
Surely. Feel free to share your thoughts in https://github.com/unjs/nitro/issues/451 but already a similar usage of current example to implement preset should be amazing start.

File: docs/content/2.guide/5.deploy/1.node-server.md [hunk 1]
```diff
@@ -49,6 +49,32 @@ module.exports = {
...
 ```
 
+### Using Node's Cluster Module
+
+Another way of starting a multi-process server with a Node.js internal module is to use the [cluster](https://nodejs.org/dist/latest/docs/api/cluster.html):
+
+```js
+import cluster from 'cluster'
+import os from 'os'
+import process from 'process'
```
Comment (stafyniaksacha) [inline docs/content/2.guide/5.deploy/1.node-server.md:59]: I suggest using `node:` protocol for built-in modules

```js
import cluster from 'node:cluster'
import os from 'node:os'
import process from 'node:process'
```js

Comment (Mastercuber) [reply docs/content/2.guide/5.deploy/1.node-server.md:59]: I will add this to the example.

Comment (Mastercuber) [reply docs/content/2.guide/5.deploy/1.node-server.md:59]: Removed it because it reads a bit better.

**Comment by pi0** [comment]
@Mastercuber I think it would be better if you put changes to nitro PR. (I can help on this if you like). This way we can include preset in next release :)

**Comment by Mastercuber** [comment]
I'm still writing at a comment. The nitro repo is forked and cloned, but I have trouble with `pnpm` or the registry I use. I will investigate this after finishing the comment. Afterwards I can create a PR with the first example code

**Comment by pi0** [comment]
New preset is avialable since nitro 0.5.1 (updating lockfile). Thanks for helping on this 🚀

**Pull request merged**